src/sentry/search/events/builder/metrics.py

```diff
@@ -185,7 +185,7 @@ def are_columns_resolved(self) -> bool:
         # are passed as aliased expressions to the MQB query transformer.
         if self.use_metrics_layer:
             first_column = self.columns[0]
-            return self.columns and (
+            return bool(self.columns) and (
```

Comment:
isn't this implied by self.columns[0] above?

Reply:
yeah true it would probably crash already from that